Salary App/Features/ValidationManager/ValidationManager.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ValidationManager.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+//
+
+import Foundation
+
+enum FormError: Error {
+    case containsEmptyString
+    case formContainsLetters
+    case invalidSalary
+    case isValidPenContribution
+}
+
+extension FormError: LocalizedError {
+    var errorLocalDescription: String {
+        switch self {
+        case .containsEmptyString:
+            return "The salary textfield contains an empty string, please enter a value"
+        case .formContainsLetters:
+            return "This textfield contains letters, please enter a number"
+        case .invalidSalary:
+            return "This is not a valid salary"
+        case .isValidPenContribution:
+            return "This is not a valid pension contribution"
+        }
+    }
+}
+
+struct ValidationManager {
+    
+    func isValid(texts: [String]) throws {
+        for text in texts {
+            if text.isEmpty {
+                throw FormError.containsEmptyString
+            }
+        }
+
+     }
+    
+    func inValidNumber(values: [String]) throws {
```

Comment:
Remove this function and do the validation in this function in the isValid function above based on my comments above.